app/views/content/life-as-a-teacher/change-careers/how-to-change-careers-to-become-a-teacher/_article.html.erb

```diff
@@ -0,0 +1,33 @@
+<div class="row inset">
+  <section class="col col-720 col-space-l-top">
+  <h2 class="heading--box-blue">What you'll need to become a teacher</h2>
+    <p>You’ll need a degree and <a href="/train-to-be-a-teacher/what-is-qts">qualified teacher status (QTS)</a> to work in the majority of schools in England. This  includes state maintained primary, secondary and special schools.</p>
+    <p>There are different ways to get QTS.</p>
+    <h3>If you have a degree</h3>
+    <p>If you already have a degree, you can do a postgraduate teacher training course to get QTS.</p>
+    <p>It doesn't matter when you graduated, and your degree subject doesn't necessarily need to be the subject you want to teach. Once you have QTS you could teach various subjects, providing you can show you have enough knowledge.</p>
+    <p>Most postgraduate teacher training courses charge fees. However, there are ways to fund your training, for example, through tuition fee and maintenance loans. You may also be eligible for a bursary or scholarship of up to <%= v :scholarships_generic_maxshortened %>.</p>
+    <p>Other postgraduate teacher training courses come with a salary. This means you will not pay tuition fees and will receive a salary while you train.</p>
+    <p><a href="/train-to-be-a-teacher/if-you-have-a-degree">Find out more about postgraduate teacher training</a>.</p>
+    <h3>If you do not have a degree</h3>
+    <p>If you do not have a degree, you can:</p>
+    <ul>
+      <li>do undergraduate teacher training to get a degree alongside qualified teacher status QTS</li>
+      <li>check if you're eligible to do a teaching degree apprenticeship</li>
+    </ul>
+    <p>Undergraduate teacher training courses can be funded through tuition fee and maintenance loans. You may also be eligible for a bursary of <%= v :bursaries_undergraduate %>.</p>
+    <p>Teaching degree apprenticeships allow you work in a school and earn a salary while getting a bachelor’s degree, so you do not have to pay tuition fees.</p>
+    <p><a href="/train-to-be-a-teacher/if-you-dont-have-a-degree">Find out more about training to teach if you do not have a degree</a>.</p>
+    <h3>If you're unable to get a degree</h3>
```

Comment:
Maybe change to 'If you do not want get a degree'? Or 'If you do not want to train for a degree'?